lib/adapter.ts

```diff
@@ -1,27 +1,30 @@
 import { Database, StorageAdapter } from "../deps.ts";
-import { SessionJson } from "./json_model.ts";
+import { Session } from "./json_model.ts";
 
 export class DenoDBAdapter<T> implements StorageAdapter<T> {
   constructor(db: Database) {
-    db.link([SessionJson]);
+    db.link([Session]);
   }
+
   async read(key: string) {
-    const session = await SessionJson.find(key);
-    return session ? JSON.parse(session.value as string) : null;
+    const session = await Session.find(key);
+    return session ? JSON.parse(session.value as string) : undefined;
```

Comment:
Does this work if the session data is of type number, and 0 is stored?

Reply:
Yes, because in db it's stored as "0", `JSON.parse` will return 0.

Reply:
Oh right! I missed that. Thanks for the explanation.